docs/building-modules/module-manager.md

```diff
@@ -135,7 +135,7 @@ The module manager is used throughout the application whenever an action on a co
 - `SetOrderEndBlockers(moduleNames ...string)`: Sets the order in which the `EndBlock()` function of each module will be called at the beginning of each block. This function is generally called from the application's main [constructor function](../basics/app-anatomy.md#constructor-function).
 - `RegisterInvariants(ir sdk.InvariantRegistry)`: Registers the [invariants](./invariants.md) of each module.
 - `RegisterRoutes(router sdk.Router, queryRouter sdk.QueryRouter, legacyQuerierCdc *codec.LegacyAmino)`: Registers module routes to the application's `router`, in order to route [`message`s](./messages-and-queries.md#messages) to the appropriate [`handler`](./handler.md), and module query routes to the application's `queryRouter`, in order to route [`queries`](./messages-and-queries.md#queries) to the appropriate [`querier`](./querier.md).
-- `RegisterQueryServices(grpcRouter grpc.Server)`: Registers all module gRPC query services.
```

Comment:
I'd leave this for now since the configurator is from #7500 and won't be included on v0.40?

Reply:
right, reverted!